charter.html

```diff
@@ -382,16 +381,15 @@ <h2>
           Success Criteria
         </h2>
         <p>
-          In order to advance to <a href=
-          "https://www.w3.org/Consortium/Process/#RecsPR" title=
-          "Proposed Recommendation">Proposed Recommendation</a>, each normative
+          In order to advance beyond <a href=
+          "https://www.w3.org/policies/process/#RecsCR">Candidate Recommendation</a>, each normative
           specification is expected to have <a href=
-          "https://www.w3.org/Consortium/Process/#implementation-experience">at
+          "https://www.w3.org/policies/process/#implementation-experience">at
           least two independent, interoperable implementations</a> of every feature defined in
-          the specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperating with each other. In order to advance to Proposed Recommendation, each normative specification must have an open test suite of every feature defined in the specification..
+          the specification, where interoperability can be verified by passing open test suites. In order to advance to Proposed Recommendation, each normative specification must have an open test suite of every feature defined in the specification..
```

Comment:
```suggestion
          the specification, where interoperability can be verified by passing open test suites.
```
Per https://w3c.github.io/charter-drafts/charter-template.html#success-criteria